README.md

```diff
@@ -5,11 +5,7 @@
 The project uses a virtual environment to isolate package dependencies. To create the virtual environment and install required packages, run the following from a bash shell terminal, in the project root directory:
 
 ```bash
-$ poetry install
-```
-Once the setup script has completed and all packages have been installed, start the Flask app by running:
-```bash
-$ poetry run flask run
+rebase$ vagrant up
 ```
 
```

Comment:
Your README doesn't currently mention your tests at all (it should), but more importantly, is it possible to run tests from vagrant? I don't see any firefox installation...
If not, don't you need to keep poetry install instructions here?

Separate to tests, do you really need to insist that the reader uses vagrant?